src/server/images/models.js

```diff
@@ -23,4 +25,38 @@ export default class ImageManager {
       cb(new Error('database error'));
     });
   }
+  static getImageUrls(params, cb) {
```

Comment:
use literal parameter for readability.
e.g.)getImageUrls({itemKey, userKey}, cb)